ang/civicase/activity/card/directives/activity-card-long.directive.html

```diff
@@ -24,44 +24,42 @@
         <!-- End - Task - Checkbox -->
         <!-- Type -->
         <span
-          class="civicase__activity-type"
-          ng-if="activity.type !== 'File Upload'"
-          ng-class="{'civicase__activity-type--completed': (activity.category.indexOf('task') > -1 && activity.is_completed)}">
-          {{ activity.type }}
+          civicase-activity-type-tooltip
```

Comment:
is this directive using the parent scope?

Reply:
Yes, true.

Reply:
Not anymore, created isolated scope.